src/annotator/integrations/test/html-test.js

```diff
@@ -178,6 +178,87 @@ describe('HTMLIntegration', () => {
         integration.fitSideBySide({ expanded: false });
         assert.deepEqual(getMargins(), [null, null]);
       });
+
+      context('main content area has margin:auto', () => {
+        function getComputedMargins(element) {
+          const leftMargin = Math.floor(
+            parseInt(window.getComputedStyle(element).marginLeft, 10)
+          );
+          const rightMargin = Math.floor(
+            parseInt(window.getComputedStyle(element).marginRight, 10)
+          );
+          return [leftMargin, rightMargin];
+        }
+
+        // Add a style node to set a max-width and auto margins on the body
+        function appendBodyStyles(document_) {
+          const el = document_.createElement('style');
+          el.type = 'text/css';
+          el.textContent = 'body { margin: 0 auto; max-width: 400px }';
+          el.classList.add('js-style-test');
+          document_.body.appendChild(el);
+        }
+
+        beforeEach(() => {
```

Comment:
You could use `before` and `after` here for changes that only need to be done once for this group of tests, rather than after every test.